bin/cut-release.js

```diff
@@ -299,6 +299,25 @@ var prompts = [
   },
   {
     type: 'confirm',
+    name: 'confirmBranch',
+    message: function (answers) {
+      var msg = 'Not on \'master\' branch; actually on branch \'' + answers.branch + '\'. Continue'
+      if (dryRun) {
+        msg += ' with dry run'
+      }
+      msg += '?'
+      return msg
+    },
+    when: function (answers) {
+      if (answers.branch === 'master') {
+        answers.confirmBranch = true
+      }
+
+      return answers.branch !== 'master'
+    }
```

Comment:
Oh, I see how the flags work now.. let me fix that. I assumed there was a `--confirm` flag that just said yes to every question.